### PR TITLE
CI / Tests

### PR DESCRIPTION
**Pull request by risktoparkrageram**
Adds tests, continuous integration, and release workflows

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dev dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools poetry
```
Comment (janrito) [inline .github/workflows/ci.yaml:31]: i think the recommendation is to use pipx to install poetry

**Review by janrito — APPROVED** [review]
awesome

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (risktoparkrageram) [reply .github/workflows/ci.yaml]: https://github.com/Beauhurst/vatvalidate/pull/2/commits/a660beb685e7e74dd9cf984897e6f45c8bbbafc0 :fire:

**Comment by risktoparkrageram** [comment]
thanks @janrito

**Pull request merged**